src/bucket.ts

```diff
@@ -3438,12 +3438,6 @@ class Bucket extends ServiceObject {
       });
     }
 
-    const contentType = mime.contentType(path.basename(pathString));
```

Comment:
Removed as this is identically covered inside of `createWriteStream()` when this code makes its way there.